src/clusterfuzz/fuzz/engine.py

```diff
@@ -44,12 +44,14 @@ class FuzzResult(object):
   """Represents a result of a fuzzing session: a list of crashes found and the
   stats generated."""
 
-  def __init__(self, logs, command, crashes, stats, time_executed):
+  def __init__(self, logs, command, crashes, stats, time_executed,
+               process_timed_out=None):
     self.logs = logs
     self.command = command
     self.crashes = crashes
     self.stats = stats
     self.time_executed = time_executed
+    self.process_timed_out = process_timed_out
```

Comment:
nit: should we just name this `timed_out` ? This is more consistent with the other flags (e.g. `time_executed` instead of `process_time_executed`).

Reply:
Probably, i just think this is more clearly distinguished from e.g. libFuzzer's timeouts

Reply:
It seems a bit inconsistent with the other flags here which are similarly talking about the process. Let's just rename it, and get this merged.

Reply:
OK